server/tests/live/test_auth.py

```diff
@@ -1062,3 +1073,4 @@ async def test_anonymous_invite(client, world, stream_room, bbb_room):
             "room:poll.vote",
             "room:poll.read",
         }
+        
```

Comment:
Please configure your editor to automatically clean trailing whitespaces (at the end of line).